pandas/tests/groupby/test_api_consistency.py

```diff
@@ -16,16 +14,13 @@
 )
 
 
-def test_frame_consistency(request, groupby_func):
+def test_frame_consistency(groupby_func):
     # GH#48028
     if groupby_func in ("first", "last"):
-        msg = "first and last are entirely different between frame and groupby"
-        request.node.add_marker(pytest.mark.xfail(reason=msg))
-    if groupby_func in ("cumcount",):
-        msg = "DataFrame has no such method"
-        request.node.add_marker(pytest.mark.xfail(reason=msg))
+        # first and last are entirely different between frame and groupby
```

Comment:
For these cases where nothing is tested, I would prefer using `pytest.skip(reason)` just so it's clear that nothing is supposed to be tested

Reply:
Makes sense - I'll do a follow up.

Reply:
Thanks (totally minor)